packages/go_router/lib/src/router.dart

```diff
@@ -152,6 +152,12 @@ class GoRouter extends ChangeNotifier with NavigatorObserver {
 
   /// Navigate to a URI location w/ optional query parameters, e.g.
   /// `/family/f2/person/p1?color=blue`
+  ///
+  /// The `extra` provides a different way to send additional data that is not
+  /// part of the location string. If the application is targeting web platform,
+  /// consider use [BrowserState] hold the data. See
+  /// [Extra parameter](https://github.com/flutter/packages/blob/main/packages/go_router/example/lib/extra_param.dart)
```

Comment:
This link will break if we rename/move the file. Do we have other options?

Reply:
not that I am aware of. we could build some validation ci to make sure all the links are valid.